components/bluetooth/lib.rs

```diff
@@ -810,25 +843,33 @@ impl BluetoothManager {
             services_vec.retain(|ref s| s.uuid == uuid);
         }
         services_vec.retain(|s| !uuid_is_blocklisted(&s.uuid, Blocklist::All));
+
+        // Step 7: If its argument is empty, throws a NotFoundError.
         if services_vec.is_empty() {
             return drop(sender.send(Err(BluetoothError::NotFound)));
         }
 
         return drop(sender.send(Ok(BluetoothResponse::GetIncludedServices(services_vec))));
     }
 
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattservice-getcharacteristic
```

Comment:
We really need to link to https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren somewhere too.